tests/unit/LHNFilterTest.js

```diff
@@ -247,17 +247,17 @@ describe('Sidebar', () => {
 
             // Given these combinations of booleans which result in the report being filtered out (not shown).
             const booleansWhichRemovesInactiveReport = [
-                // isUserCreatedPolicyRoom
-                JSON.stringify([false, false, true, false, false, false, false]),
+                // hasComments is false
```

Comment:
I think you can just remove this comment. Everything is false in this array, so no need to call out `hasComments` specifically

Reply:
👍  sounds good

Reply:
Updated!